python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -148,6 +162,36 @@ def from_xml(cls, xml: _XMLWrapper) -> "NCMECEntryUpdate":
             fingerprints={
                 x.tag: x.text for x in xml.maybe("fingerprints") if x.has_text
             },
+            feedback=(
+                [
+                    {
+                        "sentiment": x.tag,  # "affirmativeFeedback" or "negativeFeedback"
+                        "type": x.str("type"),
+                        "latest_feedback_time": x.str("lastUpdateTimestamp"),
+                        "members": [
+                            {"id": m.str("id"), "name": m.text}
+                            for m in x.maybe("members")
+                            if m.has_text
+                        ],
```

Comment:
(note to self): Should this be dict instead of synthesizing a a dict? What about a dedicated class?